README.md

```diff
@@ -154,17 +154,17 @@ $mysqli->set_charset('utf8mb4');
 
 ### Windows issue
 
-MySQL maybe crash when Host is Windows platform: 
+MySQL may crash when Host is running Windows:
 
 ```
 The Auto-extending innodb_system data file './ibdata1' is of a different size 0 pages than specified in the .cnf file
 ```
 
-You can try to fix it with [`mysql-windows.cnf`](mysql/mysql-windows.cnf)
+You can try to fix it by adding [`mysql-windows.cnf`](mysql/mysql-windows.cnf)
 ([download](https://downfile.github.io/download?url=https%3A//raw.githubusercontent.com/jakubboucek/docker-lamp-devstack/master/mysql/mysql-windows.cnf))
-and add it to path `/etc/mysql/conf.d/` inside Docker container.
+and to your MySQL config directory `/etc/mysql/conf.d/` inside the Docker container.
```

Comment:
@spaze Tady asi `Your` nedává smysl, ne? (počítám, že když něco stáhnu v Docker image, tak se na to dívám jak na blackbox, nějak mi u toho nesedí `your MySQL` a smysl dokumentace je, že když dovnitř na správné místo položíš mňanku, tak se něco stane – zde bych `your` chápal jako odkaz na instalaci MySQL na rodičovském stroji místo containeru) A ještě nechybí tam sloveso `add`, nebo jiné??

Reply:
Když už něco dáváš *dovnitř*, tak to není *blackbox* :-) Na konci tý věty to docela vysvětluješ.
Ale asi by šlo napsat i "and add it to the MySQL config directory ..."
("add" mi vypadlo, sorry!)

Reply:
Ty asi nemáš kočku, viď? Když bys měl, znal bys bezpečně koncept blackboxu, kam něco malým otvorem vložíš a pak se dějou věci. :-D

Reply:
😸